correction_test.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import pysam
+import gffutils
+
+from src.illumina_exon_corrector import IlluminaExonCorrector
+from src.alignment_info import AlignmentInfo
+from src.common import junctions_from_blocks, overlaps
+from src.short_utils import get_region_from_db
+
```

Comment:
better to use double blank lines between functions to keep with IsoQuant style guide PEP8 (but not methods)
https://peps.python.org/pep-0008/

Reply:
also, is it possible to switch to 4 spaces instead of tab?